corehq/apps/couch_sql_migration/management/commands/migrate_domain_from_couch_to_sql.py

```diff
@@ -82,12 +82,19 @@ def handle(self, domain, **options):
 
         if options['MIGRATE']:
             self.require_only_option('MIGRATE', options)
-            set_couch_sql_migration_started(domain, self.dry_run)
+
+            if options.get('run_timestamp'):
+                if not couch_sql_migration_in_progress(domain):
+                    raise CommandError("Migration must be in progress if run_timestamp is passed in")
```

Comment:
could you not resume a 'dry run'?

Reply:
You can. I guess I should make it clear in the comment, but that function should accept dry runs as well.